src/client/cli.ts

```diff
@@ -11,6 +12,7 @@ export interface BatchExplorerArgs {
 export function parseArguments(argv: string[]): BatchExplorerArgs {
     return commander
         .version(app.getVersion())
+        .option("--do-not-autoupdate", "Will not autoupdate the application")
```

Comment:
should likely make this --autoupdate <true|false> so the user can either enable or disable

Reply:
agree. Just use autoupdate, and set default to false.

Reply:
If the flag is set, that means that value is already set to true so in this case, that means autoupdate will be turned off. When the user does not set a flag, then the value is false so in this case, the user did not try to turn off autoupdate.

Reply:
Discussed offline